frontend/src/redux/create-app-store.ts

```diff
@@ -0,0 +1,25 @@
+import { applyMiddleware, combineReducers, createStore, Store } from "redux"
+import { persistStore, persistReducer, PersistConfig, Persistor } from "redux-persist"
+import storage from 'redux-persist/lib/storage'
+import thunk from "redux-thunk"
+import { composeWithDevTools } from "redux-devtools-extension"
+
+import { AppState } from "../types"
+import * as reducers from "./reducers"
+
+const persistConfig: PersistConfig<AppState> = {
```

Comment:
I'm not sure what the API for this object is--is this automatically persisting everything, or nothing?

Reply:
By default, if you do not provide either a whitelist or blacklist, everything is persisted. I can add that as a comment, but maybe the best way to include that information is in a readme in the app.

Reply:
I really think our _default_ should be to persist nothing.

Reply:
Can we make that a trackable github issue and merge in this approach now? I would like to have more eyes on that decision, and I think that would be a good way for engineering to come to a consensus on that topic, not on this specific PR.

I'm pushing back a bit on some of these comments, but it's not because I think they're a bad idea or maybe the right way to do stuff. I think what I am shooting for now is _something_ in boxcar. Then, when the baseline is set, we can iterate. Would that be okay with you for a first pass?

Reply:
YES